### PR TITLE
Export of study and patient 

### PR DESCRIPTION
**Pull request by LoicCadour**
For now, you can only export series by series, which, if you want to export a whole patient, can be quite of a pain.
This PR brings the export function to study and patient.

Also medActionTbx (the tbx in the Browser) was not updated when selecting a study, so I added a behaviour.

### CODE REVIEW

File: src/medCore/gui/database/medDatabaseView.cpp [hunk 1]
```diff
@@ -282,6 +284,7 @@ void medDatabaseView::onViewSelectedItemRequested(void)
     }
 }
 
+
```
Comment (mathildemerle) [inline src/medCore/gui/database/medDatabaseView.cpp]: small additional line to remove ^^

**Comment by mathildemerle** [comment]
Tested, it works. However, i still have this problem: https://github.com/Inria-Asclepios/music/issues/223 Do you have that on your windows?

**Comment by LoicCadour** [comment]
I am on Mac and I have this behaviour as well

**Comment by fcollot** [comment]
approved

**Pull request merged**